src/lib/perf_histograms/rpc.ml

```diff
@@ -10,6 +10,10 @@ let decorate_dispatch ~name (dispatch : ('q, 'r) Intf.dispatch) :
   Perf_histograms0.add_span
     ~name:(sprintf "rpc_dispatch_%s" name)
     (Time.diff (Time.now ()) start) ;
+  Coda_metrics.(
+    Gauge.set
+      (Network.rpc_latency_ms ~name)
+      Core.Time.(Span.to_ms @@ diff (now ()) start)) ;
```

Comment:
I thought we were saying this happens too frequently to be a Gauge, and we'd need to send the histogram to prometheus -- did I misunderstand that? cc @nholland94